src/dearpygui_commands.h

```diff
@@ -871,6 +876,26 @@ get_text_size(PyObject* self, PyObject* args, PyObject* kwargs)
 	return GetPyNone();
 }
 
+static PyObject*
+set_anti_aliasing(PyObject* self, PyObject* args, PyObject* kwargs)
```

Comment:
Since the values set by this function are global and affect the entire frame, `configure_app` looks like a proper place for them - no need to introduce one more config function, IMHO.